nucypher/crypto/kits.py

```diff
@@ -76,3 +80,78 @@ def __init__(self, *args, **kwargs) -> None:
     def from_bytes(cls, some_bytes):
         capsule, sender_pubkey_sig, ciphertext = cls.split_bytes(some_bytes)
         return cls(capsule=capsule, sender_pubkey_sig=sender_pubkey_sig, ciphertext=ciphertext)
+
+
+Revocation = namedtuple('Revocation', ['prefix', 'arrangement_id', 'signature'])
+
+
+class RevocationKit:
+
+    def __init__(self, policy_treasure_map):
+        self.revocations = dict()
+        for node_id, arrangement_id in policy_treasure_map:
+            self.revocations[node_id] = Revocation(b'REVOKE-', arrangement_id,
+                                                   constants.NOT_SIGNED)
+
+    def __iter__(self):
+        return iter(self.revocations.values())
+
+    def __getitem__(self, node_id):
+        return self.revocations[node_id]
+
+    def __len__(self):
+        return len(self.revocations)
+
+    def __eq__(self, other):
+        return self.revocations == other.revocations
+
+    @staticmethod
+    def revocation_to_bytes(revocation):
+        prefix, arrangement_id, signature = revocation
+        return b''.join((prefix, arrangement_id, bytes(signature)))
```

Comment:
... well then why is this not `bytes(revocation)`?

Reply:
Can you implement `__bytes__` on a `namedtuple`?

Reply:
Heh - when you ask yourself this kind of question - you need a class.

Reply:
It was a class before and I didn't really like it.

Reply:
You also said that you dont like it as a namedtuple.